client/my-sites/theme/main.jsx

```diff
@@ -498,17 +498,17 @@ const ThemeSheet = React.createClass( {
 
 		return (
 			<Main className="theme__sheet">
-				<QueryCanonicalTheme themeId={ this.props.id } siteId={ siteID } />
+				<QueryCanonicalTheme themeId={ this.props.id } siteId={ siteId } />
 				{ currentUserId && <QueryUserPurchases userId={ currentUserId } /> }
-				{ siteID && <QuerySitePurchases siteId={ siteID } /> }
-				{ siteID && <QuerySitePlans siteId={ siteID } /> }
+				{ siteId && <QuerySitePurchases siteId={ siteId } /> }
+				{ siteId && <QuerySitePlans siteId={ siteId } /> }
 				<DocumentHead
 					title={ title }
 					meta={ metas }
 					link={ links } />
 				<PageViewTracker path={ analyticsPath } title={ analyticsPageTitle } />
 				{ this.renderBar() }
-				{ siteID && <QueryActiveTheme siteId={ siteID } /> }
+				{ siteId && <QueryActiveTheme siteId={ siteId } /> }
```

Comment:
Same thing, we can get rid of the siteId check by instead adding a siteId check
to https://github.com/Automattic/wp-calypso/blob/master/client/components/data/query-active-theme/index.jsx#L33 and relaxing the prop requirement. Best to leave this to another PR though.